DatabaseFunctionsGenerator/CSharpGenerator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DatabaseFunctionsGenerator
+{
+    public class CSharpGenerator : IGenerator
+    {
+        private Database _database;
+
+        private CSharpModelsGenerator _cSharpModelsGenerator;
+
+        public CSharpGenerator(Database database)
+        {
+            _database = database;
+

```

Comment:
Remove this spaces here. No need for them

Reply:
I like to left some new lines between different type of code.